modules/create-wif-cluster-cli.adoc

```diff
@@ -0,0 +1,206 @@
+// Module included in the following assemblies:
+//
+// * osd_install_access_delete_cluster/creating-a-gcp-cluster-with-workload-identity-federation.adoc
+
+
+:_mod-docs-content-type: PROCEDURE
+[id="create-wif-cluster-cli_{context}"]
+= Creating a Workload Identity Federation cluster using the OCM CLI
+
+You can create an {product-title} on {GCP} cluster with Workload Identity Federation (WIF) using the OpenShift Cluster Manager CLI (`ocm`) in interactive or non-interactive mode.
+
+[IMPORTANT]
+====
+To create a WIF-enabled cluster, the OpenShift Cluster Manager CLI (`ocm`) must be version 1.0.2 or greater.
+====
+
+Before creating the cluster, you must first create a WIF configuration.
+[NOTE]
+====
+Migrating an existing non-WIF cluster to a WIF configuration is not supported. This feature can only be enabled during new cluster creation.
+====
+
+[id="create-wif-configuration_{context}"]
+== Creating a WIF configuration
+
+.Procedure
+You can create a WIF configuration using the `auto` mode or the `manual` mode.
+
+The `auto` mode enables you to automatically create the service accounts for {product-title} components as well as other IAM resources.
+
+Alternatively, you can use the `manual` mode to create a `script.sh` file in order review the details in regard to the service accounts for {product-title} components and other IAM resources before using your preferred options.
```

Comment:
In manual mode user is provided with commands (script.sh) and user needs to manually configure their cloud.